doc/duckarrays.rst

```diff
@@ -0,0 +1,54 @@
+.. currentmodule:: xarray
+
+Working with numpy-like arrays
```

Comment:
Is it worth referencing that xarray handles dask arrays natively?

Reply:
I added a note. Does that help?